### PR TITLE
Add vela robot for issue triage

### LINKED ISSUE #1806: [Feature] create github action bot for issue triage
Follow the Grafana way: https://github.com/grafana/grafana/blob/main/.github/bot.md
Using the grafana-github-actions:  https://github.com/grafana/grafana-github-actions

TBD：

* https://github.com/oam-dev/kubevela/blob/main/.github/commands.json
* https://github.com/oam-dev/kubevela/blob/main/.github/commands.json

### PR DESCRIPTION
**Pull request by wangyuan249**
**What this PR does / why we need it**:
We need a Kubevela robot for us to do some automation issue triage, duplicate issue judgement and comment guidance like this:
image

And we refer to the official implementation methods of vscode and grafana to build our robot and GitHub action trigger mechanism
**Which issue(s) this PR fixes**:

Fix #1806 

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/issue-commands.json [hunk 1]
```diff
@@ -0,0 +1,173 @@
+[
+  {
+    "type": "comment",
+    "name": "needsMoreInfo",
+    "allowUsers": [
+	    "wangyuan249",
+	    "yangsoon",
+	    "chivalryq",
+	    "wonderflow"
```
Comment (wonderflow) [inline .github/issue-commands.json:9]: 这个名单不对，要根据maintainer名单来。 另外这个下面很多重复，是不是可以用一份，或者引用别的地址？

File: .github/workflows/pr-commands.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        with:
+          repository: "oam-dev/kubevela-github-actions"
+          path: ./actions
+          ref: main
+      - name: Install Actions
+        run: npm install --production --prefix ./actions
+      - name: Run Commands
+        uses: ./actions/commands
+        with:
+          token: ${{secrets.VELA_BOT_TOKEN}}
```
Comment (wonderflow) [inline .github/workflows/pr-commands.yml:23]: is the token ready now?

File: .github/workflows/issue-commands.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        with:
+          repository: "oam-dev/kubevela-github-actions"
+          path: ./actions
+          ref: main
+      - name: Install Actions
+        run: npm install --production --prefix ./actions
+      - name: Run Commands
+        uses: ./actions/commands
+        with:
+          token: ${{secrets.VELA_BOT_TOKEN}}
```
Comment (wonderflow) [inline .github/workflows/issue-commands.yml:23]: is the token ready now?

File: ISSUE_TRIAGE.md [hunk 4]
```diff
@@ -315,9 +315,10 @@ please provide as much information as possible.
...
-Many of these automated behaviors are defined in [commands.json](https://github.com/oam-dev/kubevela/blob/main/.github/commands.json).
+Many of these automated behaviors are defined in [issue-commands.json](https://github.com/oam-dev/kubevela/blob/main/.github/issue-commands.json).
 
-* Add /duplicate `#<issue number>`  to have KubeVela label & close issue with an appropriate message.
-* Add `bot/question` and the bot will close it with an appropriate message.
+* Add comment `\duplicate` to have `type/duplicate` label.
+* Add label `bot/duplicate` to have `type/duplicate` label & the bot will close issue with an appropriate message.
+* Add label `bot/no new info` and the bot will close it with an appropriate message.
 
 [Read more on bot actions](https://github.com/oam-dev/kubevela/blob/main/.github/bot.md)
```
Comment (wonderflow) [inline ISSUE_TRIAGE.md:324]: do we have this readme now?

File: ISSUE_TRIAGE.md [hunk 5]
```diff
@@ -315,9 +315,10 @@ please provide as much information as possible.
 ## Automation
 
 We have some automation that triggers on comments or labels being added to issues.
-Many of these automated behaviors are defined in [commands.json](https://github.com/oam-dev/kubevela/blob/main/.github/commands.json).
+Many of these automated behaviors are defined in [issue-commands.json](https://github.com/oam-dev/kubevela/blob/main/.github/issue-commands.json).
 
-* Add /duplicate `#<issue number>`  to have KubeVela label & close issue with an appropriate message.
-* Add `bot/question` and the bot will close it with an appropriate message.
+* Add comment `\duplicate` to have `type/duplicate` label.
```
Comment (wonderflow) [inline ISSUE_TRIAGE.md:320]: please make sure the command `/duplicate` or `\duplicate`

File: ISSUE_TRIAGE.md [hunk 6]
```diff
@@ -315,9 +315,10 @@ please provide as much information as possible.
...
+* Add label `bot/duplicate` to have `type/duplicate` label & the bot will close issue with an appropriate message.
+* Add label `bot/no new info` and the bot will close it with an appropriate message.
```
Comment (wonderflow) [inline ISSUE_TRIAGE.md:322]: why the lable `bot/no new info` leads to close?

File: .github/issue-commands.json [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+    ],
+    "action": "updateLabels",
+    "addLabel": "needs more info"
```
Comment (wonderflow) [inline .github/issue-commands.json:12]: 这些label是不是我们repo 已经都存在？并且跟已经存在的一致？

File: .github/issue-commands.json [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+    "action": "updateLabels",
+    "addLabel": "type/duplicate"
+  },
+  {
+    "type": "label",
+    "name": "bot/duplicate",
+    "addLabel": "type/duplicate",
+    "removeLabel": "bot/duplicate",
+    "action": "close",
+    "comment": "Thanks for creating this issue! It looks like this has already been reported by another user. We’ve closed this in favor of the existing one. Please consider adding any details you think is missing to that issue.\n\nTo avoid having your issue closed in the future, please read our [CONTRIBUTING](https://github.com/oam-dev/kubevela/blob/master/CONTRIBUTING.md) guidelines.\n\nHappy graphing!"
```
Comment (wonderflow) [inline .github/issue-commands.json:158]: 这里要关闭重复的issue，能把已经存在的issue号记录在comment里吗？

File: .github/issue-commands.json (hunk 7 above)
Comment (wangyuan249) [reply .github/issue-commands.json:12]: 一致

File: .github/issue-commands.json (hunk 8 above)
Comment (wangyuan249) [reply .github/issue-commands.json:213]: 目前暂时处理为 要求 comment duplicate的人必须加 #issue number   后续会完善  实现读入duplicate 后面的参数(有看到源码中对args的处理)

File: .github/comment-userList.md [hunk 9]
```diff
@@ -0,0 +1,5 @@
+BinaryHB0916 <br />
+wangyuan249  <br />
+yangsoon     <br />
+chivalryq    <br />
+
```
Comment (wonderflow) [inline .github/comment-userList.md:5]: comment.userlist

File: .github/issue-commands.json [hunk 10]
```diff
@@ -0,0 +1,191 @@
+[
+  {
+    "type": "comment",
+    "name": "needsMoreInfo",
+    "allowUsers": [
+	    "wangyuan249"
```
Comment (wonderflow) [inline .github/issue-commands.json:6]: remove the special

File: .github/pr-commands.json [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+    "action": "updateLabel",
+    "addLabel": "change/monitorfile"
+  },
+  {
+    "type": "changedfiles",
+    "matches": ".github/**/*",
+    "action": "updateLabel",
+    "addLabel": "change/.github"
+  }
+]
```
Comment (wonderflow) [inline .github/pr-commands.json:16]: no need

File: .github/workflows/issue-commands.yml [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+    types: [created]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+        with:
+          repository: "oam-dev/kubevela-github-actions"
```
Comment (wonderflow) [inline .github/workflows/issue-commands.yml:15]: make a release

File: .github/workflows/pr-commands.yml [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+          configPath: pr-commands
```
Comment (wonderflow) [inline .github/workflows/pr-commands.yml:24]: no need

**Review by wonderflow — APPROVED** [review]

**Pull request merged**